### DISCUSSION TITLE
Any plans for proper streaming?

### DISCUSSION THREAD

**Discussion by melluh** [blakeblackshear/frigate#948] [Ideas]
Are there any plans to implement a proper way of viewing the camera streams through the web app? While the current system does work fine in most cases, it isn't a very efficient solution, because it's opening a new TCP connection 5 times a second (or whatever your FPS is set to). Maybe something is possible with the MJPEG or RTMP stream already available in the API? Not entirely sure what's possible here.

**Comment by paularmstrong** [comment]
Yes: https://github.com/blakeblackshear/frigate/issues/422
